jobs/async-upload/job/config.py

```diff
@@ -236,10 +244,11 @@ def get_config(argv: list[str] | None = None) -> Dict[str, Any]:
                 "region": None,
                 "access_key_id": None,
                 "secret_access_key": None,
-                "endpoint": None,
+                "endpoint_url": None,
```

Comment:
This is a correction on the official S3 env vars.

https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-envvars.html

And so, `AWS_ENDPOINT_URL` -> `endpoint_url`